webapp/src/main/java/org/mediacloud/cliff/servlet/dto/Mention.java

```diff
@@ -0,0 +1,36 @@
+package org.mediacloud.cliff.servlet.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
+public class Mention {
+    private String countryCode;
+    private String stateCode;
+    private String county;
+    private String city;
+    private String place;
+    private Double lat;
+    private Double lon;
+    private int id;
+
+    private MentionSource mentionSource;
+
+    private String featureCode;
+    private String featureClass;
+
+
+    @AllArgsConstructor
+    @NoArgsConstructor
+    @Builder
+    @Data
+    public static class MentionSource {
+        private int charIndex;
```

Comment:
We don't really need the charIndex as we won't be saving it, not sure if we can leave it out already or if we want to take it out in enricher

Reply:
Is the FE planning to highlight the match? If that's the case, they might need it.

Reply:
Let's discuss it with Hernan just in case, but I think it's not really needed and they can manually just find the match inside the String